libcsm/sls/api.py

```diff
@@ -74,8 +74,8 @@ def get_xname(self, hostname: str) -> str:
                     return node['Xname']
             except KeyError as error:
                 raise KeyError(f'ERROR [ExtraProperties][Aliases] was not in the \
-                response from sls. These fields are expected in the json response. \
-                The resonponse was {components_response.json()}') from error
+response from sls. These fields are expected in the json response. \
+The resonponse was {components_response.json()}') from error
```

Comment:
Response here too.

Also what happens if the response.json() call fails in the exception handler or throws its own exception?

Reply:
Question for @leliasen-hpe

Reply:
I addressed this by using a try-except statement and catching a JSONDecodeError. I think this is the only exception thrown by .json(). Does that seem sufficient @mitchty @rustydb ?

Reply:
I changed this to ValueError. JSONDecodeError inherits from ValueError so this will correctly catch the error.